plugins/catalog/src/apis/EntityPresentationApi/defaults.tsx

```diff
@@ -38,13 +38,13 @@ export const UNKNOWN_KIND_ICON: IconComponent = HelpIcon;
 export const DEFAULT_ICONS: Record<string, IconComponent> = {
   api: ExtensionIcon,
   component: MemoryIcon,
-  system: BusinessIcon,
+  system: CategoryIcon,
   resource: WorkIcon,
   domain: ApartmentIcon,
   location: LocationOnIcon,
   user: PersonIcon,
   group: PeopleIcon,
-  template: LibraryAddIcon,
+  template: FeaturedPlayListIcon,
```

Comment:
While this is correct as I requested I don't think I ever noticed that this was the actual default for Templates. I'd like to see what the Maintainers think but personally using `AddCircleOutline` makes more sense as that's what is used in the sidebar:

![image](https://github.com/backstage/backstage/assets/67169551/3a53fcb2-71b5-4d50-9b68-eedf81542256)

That seems more likely to be what someone associates with Templates, again my personal opinion here ;)